mate-base/mate-session-manager/mate-session-manager-1.27.0.ebuild

```diff
@@ -19,9 +17,6 @@ LICENSE="GPL-2+ GPL-3+ HPND LGPL-2+ LGPL-2.1+"
 SLOT="0"
 IUSE="debug elogind gles2 gnome-keyring nls systemd"
 
-REQUIRED_USE="^^ ( elogind systemd )"
-
-# x11-misc/xdg-user-dirs{,-gtk} are needed to create the various XDG_*_DIRs, and
```

Comment:
?

Reply:
Ignoring I messed up the edit...

This was reported and later tested by myself that you don't need to set to elogind so I originally  honored the bug  request opened to remove it, however now I follow #mate on  libera.chat I've found this causes a random issue in the date and time settings to occur so this will be reverted anyway based on this while I follow that bug.

Reply:
I meant removing the comment.

Reply:
A case of less haste more speed unfortunately.